src/CMakeLists.txt

```diff
@@ -69,6 +66,15 @@ if(FLB_STATS)
     )
 endif()
 
+if(FLB_BUFFERING)
```

Comment:
Thank you for your reviewing.

> the pre-processor will make to skip the code.
In this case(ifdef at source code), almost blank flb_buffer*.c file is pre-processed and compiled and it will take some time.
My patch (ifdef at CMakeLists.txt) is to skip them.